resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
The main problem is that `Varchar` is not a valid column type in MySQL if it doesn't include a character limit, so this migration as it's written will fail to run on MySQL.  I'm simply trying to fix that problem.

If you want to bump it up to 255 then that's fine, but it's a common and natural approach to set column types for varchars to include a max length.